flow/raster_cache.cc

```diff
@@ -211,10 +190,10 @@ bool RasterCache::Prepare(GrContext* context,
   PictureRasterCacheKey cache_key(picture->uniqueID(), transformation_matrix);
 
   Entry& entry = picture_cache_[cache_key];
-  entry.access_count = ClampSize(entry.access_count + 1, 0, access_threshold_);
+  entry.access_count++;
   entry.used_this_frame = true;
 
-  if (entry.access_count < access_threshold_ || access_threshold_ == 0) {
+  if (entry.access_count < access_threshold_) {
```

Comment:
`access_threshold_ == 0` means never cache. This is why the unit test is failing.

Reply:
Sorry, I had to change the samentics of access_threshold. It wasn't documented before and interpreting access_threshold == 0 as "cache disabled" just breaks me.

I updated the test to reflect the saner semantics, previously they were actually asserting the wrong thing, i.e. number of prepares instead of numbers of accesses. The commit message of my latest commit has more context.

Make sure that the latest defaults seem sensible especially since users cannot change them.

Thanks

Reply:
Semantically, agreed that `access_threshold` is currently conflating two things:
   1. the number of accesses below which we don't cache
   2. whether or not to cache at all
By the semantics of (1), zero should really imply "always cache". 

Since that's not strictly related to the fix for the fact that we're failing to count accesses via `Get()`, I'd prefer we break that out into a separate change, since that's a lower-priority fix. We can then decided whether we want a replacement for the semantics of (2) above (a means to simply disable raster caching altogether.

Reply:
Sure thing, I restored the behavior and re-introduced the test. FWIW, should be fairly easy to revert as a single CL.